cs3api4lab/api/cs3apismanager.py

```diff
@@ -15,7 +15,7 @@
 from cs3api4lab.utils.file_utils import FileUtils
 from cs3api4lab.api.share_api_facade import ShareAPIFacade
 from cs3api4lab.utils.model_utils import ModelUtils
-
+from cs3api4lab.utils.asyncify import asyncify
 
 class CS3APIsManager(ContentsManager):
```

Comment:
When I reported the issue, I didn't necessarily see any issues with these requests (contents manager), if I'm not mistaken, but with the calls to our new handlers (for the cs3 apis). So I'm not sure this is necessary here, but we can leave it to try.

Reply:
It helps with reading directories, opening files etc